canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx

```diff
@@ -109,25 +110,26 @@ class TextfieldControl extends React.Component {
 
 		let display = textInput;
 		if (this.props.tableControl) {
-			const tooltipId = uuid4() + "-tooltip-column-" + this.props.propertyId.toString();
+			let content = textInput;
 			let disabled = true;
 			if (value && this.props.state !== STATES.DISABLED) {
 				disabled = false;
 			}
-			const tooltip = (
-				<div className="properties-tooltips">
-					{String(value)}
-				</div>
+			if (this.props.control.icon) {
+				content = (
+					<div className={"properties-field-textfield"}>
+						<div className={"properties-field-type-icon"}>
+							<Icon type={this.props.control.icon} />
+						</div>
+						{textInput}
+					</div>);
+			}
+			display = (
+				<TruncatedContentTooltip
+					content={content}
+					disabled={disabled}
```

Comment:
You're missing the `tooltipText` that is marked as required in this component
![image](https://user-images.githubusercontent.com/1628100/223206409-4eb32e6d-06b8-4c30-9945-97324d409bc2.png)